File: CSharp/FundamentalsPartTwo/Arrays/1. ArrayOf20Integers/ArrayOf20Integers.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

class ArrayOf20Integers
{
    static void AssignArray(int[] arr)
    {
        for (int i = 1; i < arr.Length; i++)
        {
            arr[i] = i * 5;
        }
    }
    static void WriteArray(int[] arr)
    {
        for (int i = 1; i < arr.Length; i++)
        {
            Console.WriteLine("arr[{0}]= " + arr[i], i);
        }
    }
    static void Main()
    {
        int[] array = new int[21];
        AssignArray(array);
        WriteArray(array);
    }
}

File: Javascript/Fundamentals/Arrays/Arrays/ArrayOf20Integers/scripts/ArrayOf20Integers.js

var arr = new Array(21);
for (var i = 1; i < arr.length; i++) 
{
    arr[i-1] = i + 5;
}
for (var i = 0; i < arr.length-1 ; i++) 
{
    jsConsole.writeLine(i+1 + ". " + arr[i]);
}